src/test/java/com/marmelab/quoridor/web/QuoridorEndpointTest.java

```diff
@@ -32,8 +32,11 @@ public void gameWithBoardAndPawn() throws Exception {
 
         String content = quoridor.getResponse().getContentAsString();
         assertThat(content)
-                .contains("<div class=\"pawn\" style=\"top: 160px; left: 0px\">")
-                .contains("<div class=\"square\" style=\"top: 0px; left: 80px\"></div>");
+                .contains(" <div class=\"pawn\"\n" +
```

Comment:
Testing the html result probably isn't the best choice. This gonna break every time you touch templates. You better just test what it says it tests : the presence of the pawn.